docs/src/tutorials/hybrid-applications-with-webpack/using-your-library.md

```diff
@@ -0,0 +1,85 @@
+# Run The Code
+
+The Rust Webpack template is designed for creating monorepo-style Web applications with
+Rust-generated WebAssembly and Webpack without publishing your wasm to NPM.
+This portion of the tutorial will explain how to build a [Webpack] JavaScript project
+that will run your WebAssembly code in the browser.
+
+[Webpack]: https://webpack.js.org/
+
+## Scaffold a JavaScript Project
```

Comment:
The docs here look like they may be copy/pasted and/or duplicates of what's already been extracted to dedicated docs above?